src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -539,7 +674,62 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
+                        <div><span>          cml runner \</span></div>
+                        <div><span>              --cloud=aws \</span></div>
+                        <div><span>              --cloud-region=us-west \</span></div>
+                        <div><span>              --cloud-type=m5.2xlarge \</span></div>
+                        <div><span>              --cloud-spot \</span></div>
+                        <div><span>              --labels=cml</span></div>
+                      </Tooltip>
+                      <div><span>  - step:</span></div>
+                      <div><span>      name: run</span></div>
+                      <Tooltip type="runner">
+                        <div><span>      runs-on: [self.hosted, cml]</span></div>
+                      </Tooltip>
+                      <div><span>      image: iterativeai/cml:0-dvc2-base1</span></div>
+                      <div><span>      script:</span></div>
+                      <Tooltip type="dependencies">
+                        <div><span>      - apt-get update -y</span></div>
+                        <div><span>      - apt install imagemagick -y</span></div>
+                        <div><span>      - pip install -r requirements.txt</span></div>
+                      </Tooltip>
+                      <div><span>      - git fetch --prune</span></div>
+                      <div><span>      - dvc repro</span></div>
+                      <Tooltip type="reports">
+                        <div><span>      - echo &quot;# Style transfer&quot; &gt;&gt; report.md</span></div>
+                        <div><span>      - git show origin/master:final_owl.png &gt; master_owl.png</span></div>
+                        <div><span>      - convert +append final_owl.png master_owl.png out.png</span></div>
+                        <div><span>      - convert out.png -resize 75%  out_shrink.png</span></div>
+                        <div><span>      - echo &quot;### Workspace vs. Main&quot; &gt;&gt; report.md</span></div>
+                        <div><span>      - cml publish out_shrink.png --md --title &#x27;compare&#x27; &gt;&gt; report.md</span></div>
+                        <div><span>      - echo &quot;## Training metrics&quot; &gt;&gt; report.md</span></div>
+                        <div><span>      - dvc params diff master --show-md &gt;&gt; report.md</span></div>
+                        <div><span>      - echo &gt;&gt; report.md</span></div>
+                        <div><span>      - cml send-comment report.md</span></div>
+                      </Tooltip>
+                    </Code>
+                    <ExampleBox title="CML Report">
+                      <a target="_blank" rel="noreferrer" href="https://bitbucket.org/iterative-ai/cml-cloud-case">
+                        <Image src="/img/bitbucket/cloud-report.png" alt="Bitbucket Cloud report example" />
+                      </a>
+                    </ExampleBox>
                   </Collapser>
                 )}
```

Comment:
GPU is not supported. Instead of providing this example, it might be better to disable the tab for Bitbucket + GPU.

Reply:
I did that previously but I think it looks more coherent this way and we can advert that we are not supporting GPU

Reply:
Maybe leave the tab enabled but replace the contents with a big, yellow warning

Reply:
Yes, though I'd prefer @dacbd insert a TPI BB GPU snippet instead.

Reply:
Resolving in favor of https://github.com/iterative/cml.dev/pull/362#discussion_r1014527509